.github/workflows/stale.yml

```
@@ -0,0 +1,57 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Stale Action"
+        uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          # Apply the `stale` label to an issue with no activity for 90 days.
+          days-before-issue-stale: 90
+          # Close the issue if no activity occurs within 30 days after being
+          # marked as stale.
+          days-before-issue-close: 30
```

I think the good variable names make these comments redundant

```suggestion
          days-before-issue-stale: 90
          days-before-issue-close: 30
```